### PR TITLE
search 7.3 | upgrading elasticsearch | LRDOCS-7788

### PR DESCRIPTION
**Pull request by rbohl**
Not yet tested. Please let me know how this is shaping up as an upgrade guide.
https://issues.liferay.com/browse/LRDOCS-7788

### CODE REVIEW

**Comment by rbohl** [comment]
I think the conflict is just because your fork is not up-to-date. See the [comparison of my branch with liferay-learn master](https://github.com/liferay/liferay-learn/compare/master...rbohl:search_7.3_upgrading-elasticsearch_LRDOCS-7788)

**Comment by lipusz** [comment]
Updated.

**Comment by lipusz** [comment]
**Started reviewing...** :mag::nerd_face:

**Comment by rbohl** [comment]
Elasticsearch landing page:
lrdocs-7788_eslangingpage

Upgrading Elasticsearch landing page:
lrdocs-7788_upgradelandingpage

File: docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Upgrading Search for Liferay 7.3
+
+Liferay 7.3 supports only Elasticsearch 7.9.0+ via the out-of-the-box Liferay Connector to Elasticsearch. The matrix of pre-upgrade stacks you are migrating from is more numerous, and it's important to understand the high-level steps required to safely navigate from your existing stack to the Liferay 7.3 stack.  The most basic scenario includes
```
Comment (lipusz) [inline docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md:3]: Change to _out-of-the-box Liferay Connector to Elasticsearch **7**_

File: docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+- Backing up your indexes
+- Installing Elasticsearch 7.9 
+- Connecting Liferay to Elasticsearch
+- Re-indexing the Workflow Metrics, Company, and System indexes from their dedicated user interfaces
+- Restoring the search tuning indexes from the backed up version
+
```
Comment (lipusz) [inline docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md:10]: Insert a header here, something like _Overview_

File: docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+That's an overview of the search upgrade picture, and doesn't cover any more complicated scenarios (like if you have Liferay Enterprise Search modules to upgrade as well). Find the scenario that matches your Liferay version, LES version (if using LES), and your current search engine stack. Use the *Upgrade Steps* column to guide the upgrade.
+
+| Pre-Upgrade Liferay Version [+ LES Version] | Pre-Upgrade Search Engine | Upgrade Steps |
```
Comment (lipusz) [inline docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md:13]: >Pre-Upgrade

I found it confusing. First I thought it means I should upgrade my search engine to those versions. Can you @rbohl think of a different column header?

Comment (lipusz) [reply docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md:13]: _Upgrading from DXP Version_ and _Upgrading from Elasticsearch Version_ sound more clear to me.

**Comment by lipusz** [comment]
Hey Russ,

Is it intentional that you don't link from https://github.com/lipusz/liferay-learn/blob/pr-16-LRDOCS-7788/docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-search-for-liferay-73.md#common-upgrade-steps 

to

https://github.com/lipusz/liferay-learn/blob/pr-16-LRDOCS-7788/docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-to-elasticsearch-79.md?

I think it should be mentioned in the Common Upgrade steps and in the LES steps too. :thinking:

File: docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-to-elasticsearch-79.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+```important::
+   Before upgrading Elasticsearch, back up your existing data. If something goes wrong during or after the upgrade, roll back to the previous version using the uncorrupted index snapshots. Follow the steps in `Backing up Elasticsearch <./backing-up-elasticsearch.md>`__.
+```
+
+To upgrade an existing Elasticsearch server (or cluster) to Elasticsearch 7.9+,
+
+1. [Back up the Liferay Company and System Indexes](./backing-up-elasticsearch.md).
+
+1. [Back up the application specific indexes for Search Tuning](#backing-up-and-restoring-search-tuning-indexes) (Synonym Sets and Result Rankings).
```
Comment (lipusz) [inline docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-to-elasticsearch-79.md:13]: @rbohl This does not point to anywhere, I don't see this content on the page.

Comment (lipusz) [reply docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-to-elasticsearch-79.md:13]: Ah, you meant to link to https://github.com/lipusz/liferay-learn/blob/pr-16-LRDOCS-7788/docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/backing-up-elasticsearch.md#backing-up-and-restoring-search-tuning-indexes

Comment (rbohl) [reply docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-to-elasticsearch-79.md:13]: yeah, i re-organized it and didn't change the link

**Comment by lipusz** [comment]
See https://github.com/rbohl/liferay-learn/pull/54

**Pull request closed without merging**